cmd/skuba/cluster/check.go

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2019 SUSE LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package cluster
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/SUSE/skuba/pkg/skuba"
+	"github.com/rikatz/kubepug/lib"
+	"github.com/rikatz/kubepug/pkg/formatter"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/klog"
+)
+
+type checkOptions struct {
+	K8sVersion      string
+	ForceDownload   bool
+	APIWalk         bool
+	SwaggerDir      string
+	ShowDescription bool
+}
+
+var (
+	format    string
+	filename  string
+	inputFile string
+)
+
+// NewCheckCmd creates a new `skuba check` cobra command
+func NewCheckCmd() *cobra.Command {
+	checkOptions := &checkOptions{}
+
+	cmd := &cobra.Command{
+		Use:   "check k8s-version=<version> swaggerDir=<directory> --api-walk=<true|fasle>",
```

Comment:
Suggestion to use `--kubernetes-version` to align with `skuba cluster init` optional flag.

Reply:
+1